data/cameras.xml

```diff
@@ -12742,12 +12742,12 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
-	<Camera make="SAMSUNG" model="EK-GN120" decoder_version="3" supported="no-samples">
-		<ID make="Samsung" model="EK-GN120">Samsung EK-GN120</ID>
+	<Camera make="SAMSUNG" model="EK-GN120" decoder_version="3">
+		<ID make="Samsung" model="EK-GN120">Samsung Galaxy NX</ID>
```

Comment:
```suggestion
		<ID make="Samsung" model="EK-GN120">Samsung NX U</ID>
```

This actually comes from UniqueCameraModel after Adobe DNG Conversion.

Reply:
I've spent quite some time studying Samsung NX and the models created by them, and I never encountered a "Samsung NX U". There is an [April's Fool "NX Ultra" joke](https://www.digitalcameraworld.com/news/samsung-nx-ultra-samsung-resurrects-camera-line-with-108mp-s20-ultra-sensor), but the "NX U" only seems to exist in Adobe's DNG and websites that use it for listing the cameras they support. The official name is [Samsung Galaxy NX](https://www.samsungmobilepress.com/press-releases/introducing-galaxy-nx-the-first-interchangeable-lens-camera-with-3g-4g-lte-wi-fi-connectivity) and there were two internal model numbers: EK-GN100 and EK-GN120.

Reply:
I don't understand this `<ID>` name remangling, and there's no docs.
My best guess is that indeed, the idea is to really be 1:1 with ADC.

Reply:
I understand the urge to be consistent, but quite obviously Adobe is in a dead-end here. I'd really appreciate showing the name that's actually printed on the device ("Galaxy NX"), or at least the internal model number ("EK-GN120", I'm pretty sure it's part of the fineprint on the body label, will check on Saturday). The ADC name will most probably only confuse the users.

Reply:
Maybe I'm mistaking the ID field, and it's not supposed to contain the human readable name of the camera?

Reply:
> Maybe I'm mistaking the ID field, and it's not supposed to contain the human readable name of the camera?

Yes, but at least in rawspeed tools and darktable, it is constructed from `make` + `model` ID attributes, not the ID value. 🤷 

In thi case it only helps converting "SAMSUNG" -> "Samsung", but you'll find more useful example pairs in the file.